### PR TITLE
Fix float precision in comparison

### PR DESCRIPTION
**Pull request by Aiee**
As title.
Solve the problem by a naive approach: change the epsilon to a smaller value.

### CODE REVIEW

File: tests/tck/features/lookup/LookUp.IntVid.feature [hunk 1]
```diff
@@ -477,7 +473,6 @@ Feature: LookUpTest_Vid_Int
       """
     Then the result should be, in any order:
       | SrcVID | DstVID | Ranking |
-    # FIXME(aiee): should contain 220->223
     When executing query:
       """
       LOOKUP ON lookup_edge_2 WHERE lookup_edge_2.col3 >= 100.5
```
Comment (czpmango) [inline tests/tck/features/lookup/LookUp.IntVid.feature:478]: yield col3.

Comment (Aiee) [reply tests/tck/features/lookup/LookUp.IntVid.feature]: Fixed.

**Review by CPWstatic — APPROVED** [review]

File: tests/tck/features/lookup/LookUp.feature [hunk 2]
```diff
@@ -462,14 +465,14 @@ Feature: LookUpTest_Vid_String
...
-        lookup_tag_2(col1, col2, col3, col4)
+      lookup_tag_2(col1, col2, col3, col4)
       VALUES
-        "220":(true, 100, 100.5, true),
-        "221":(true, 200, 200.5, true),
-        "222":(true, 300, 300.5, true),
-        "223":(true, 400, 400.5, true),
-        "224":(true, 500, 500.5, true),
-        "225":(true, 600, 600.5, true)
+      "220":(true, 100, 100.5, true),
```
Comment (Shylock-Hg) [inline tests/tck/features/lookup/LookUp.feature:470]: Why change this?

Comment (Aiee) [reply tests/tck/features/lookup/LookUp.feature:470]: This is changed by auto formating.

**Review by yixinglu — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**